kuma/core/fields.py

```diff
@@ -9,3 +11,33 @@ def __init__(self, max_length=7, default=settings.LANGUAGE_CODE,
         return super(LocaleField, self).__init__(
             max_length=max_length, default=default, choices=choices,
             *args, **kwargs)
+
+
+class JSONField(models.TextField):
+    """JSONField is a generic textfield that neatly serializes/unserializes
+    JSON objects seamlessly
+    see: http://djangosnippets.org/snippets/1478/
+    """
+
+    # Used so to_python() is called
+    __metaclass__ = models.SubfieldBase
+
+    def to_python(self, value):
+        """Convert our string value to JSON after we load it from the DB"""
+        if not value:
+            return dict()
+        try:
+            if (isinstance(value, basestring) or
+                    type(value) is unicode):
+                return json.loads(value)
+        except ValueError:
+            return dict()
+        return value
+
+    def get_db_prep_save(self, value, connection):
+        """Convert our JSON object to a string before we save"""
+        if not value:
```

Comment:
@robhudson Meh, yet another random utils package.
@groovecoder I thought about it but eventually didn't go that route since I wasn't sure if the schema and data was truly compatible. Let's do this in a separate step. The package is also on PyPI: https://pypi.python.org/pypi/jsonfield